src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
I think we didn't do the triage process properly here. This is not worth backporting according to our backporting rules. It affects a really low number of users but still it keeps us busy. We could be working on higher prio items. I vote for cancelling all efforts here